blips/blip-3-peerreview.md

```diff
@@ -0,0 +1,53 @@
+This BLIP is a draft: the collaborative document we use to write it is https://hackmd.io/QtoxlUimQNSJByIRKm0IwQ?edit
+
+# Peer Review
+
+| BLIP:     | 3                                                        |
+| -------- | ------------------------------------------------------------ |
+| Title:   | Peer Review                                      |
+| Owner(s):  |                                                            |
+| Author(s):  | Antonio Tenorio Fornés (antonio@decentralized.science), Kaan Uzdogan (uzdogan@mpdl.mpg.de), Paulo Colombo (pcolombo@ucm.es), Elena Pérez Tirador (elena@decentralized.science)                                                           |
+| Status:  | ![Draft](http://rfc.unprotocols.org/spec:2/COSS/draft.svg) |
+| Created: | 2020-06-09                                                   |
+| License: | BSD-2-Clause                                                 |
+
+A draft for a standardized peer review format
+
+## Standard Peer Review registries
+Currently, peer-review-app is registering many fields of a peer review such as author of the review, or the ID of the reviewed article.
+
+This standard should specify which fields to register in every record, the format and standards those fields should adopt, which are mandatory among others.
+
+
+### Peer Review Records' fields
+> [name=Paulo] What is the `id` field and why do we need it ?
+
+| Field | data type | Format/s | Mandatory |  Comments|
+| -------- | ---- | ---- | -------- |----|
+|     id | string  | IPFS Hash??   | Yes     | The id should be unique. How to make sure reviews are unique? See [Zooko's triangle](https://en.wikipedia.org/wiki/Zooko%27s_triangle) for a relevant discussion on the compromises between decentralization, human usability and unique identifiers   |
+|     timestamp | uint32     | unix timestamp     |  Yes    |
+|     author | address     |      |  Yes    |
+|     journalId | string     |      |  Yes    |
+|     publisher | string     |      |  Yes    |
+|     manuscriptId | string     |     |  Yes    |
+|     manuscriptHash | string     |     |  Yes    |
+|     recommendation | enum     |     |  Yes    |
+|     url | string     |     |  Yes    | The url should give public access to the content of the review. Alternativelly, it can be a IPFS address so the content is served from a decentralized network
+|     verified | bool     |     |  Yes    |
+|     vouchers | address[]     |     |  Yes    |
+|     vouchersMap | mapping(address => bool)     |     |  Yes    |
+
+
+- [ ] **TODO** Decide which field could be private. For instance, the review content could be publicly shared or not depending on the reviewer's preferences.
```

Comment:
Private in the sense of visibility in the contract code? Data itself will still be more or less public. AFAIK we can only tackle real privacy here by encryption or sharing data off-chain.

Reply:
Private in the sense of openly shared in a blockchain or not shared in a blockchain (but maybe yes with specific apps, service providers, etc)

I am thinking mainly about two content that could be either shared publicly or only with some parties:

- email addresses
- Peer review report content

But I am sure there are many other cases.

Taking inspiration by what Orcid does, they only share a URL of the review, then that URL can openly share the authors identity, the content of the review or neither. see https://support.orcid.org/hc/en-us/articles/360006971333-Peer-Review

Reply:
I agree with this, just want to be sure we are on the same page when we say, private means it has to be stored off-chain. This is slightly touching the SSI domain.